bluehost.php

```diff
@@ -1,33 +1,55 @@
 <?php
 /**
- * Plugin Name: MOJO Marketplace
- * Description: This plugin adds shortcodes, widgets, and themes to your WordPress site.
- * Version: 1.4.2
- * Author: Mike Hansen
- * Author URI: http://mikehansen.me?utm_campaign=plugin&utm_source=mojo_wp_plugin
+ * Plugin Name: Bluehost
+ * Description: This plugin integrates your WordPress site with the Bluehost control panel, including performance, security, and update features.
+ * Version: 1.4.3
+ * Author: Bluehost
+ * Author URI: https://www.bluehost.com/
  * License: GPLv2 or later
  * License URI: http://www.gnu.org/licenses/gpl-2.0.html
  *
- * @package Mojo Marketplace
+ * @package Bluehost
  */
 
 // Do not access file directly!
 if ( ! defined( 'WPINC' ) ) {
 	die;
 }
 
-define( 'MM_VERSION', '1.4.2' );
+// Define constants
+define( 'MM_VERSION', '1.4.3' );
 define( 'MM_BASE_DIR', plugin_dir_path( __FILE__ ) );
 define( 'MM_BASE_URL', plugin_dir_url( __FILE__ ) );
 define( 'MM_ASSETS_URL', 'https://www.mojomarketplace.com/mojo-plugin-assets/' );
 
+define( 'BLUEHOST_PLUGIN_VERSION', MM_VERSION );
+
 // Composer autoloader
 if ( version_compare( phpversion(), 5.3, '<' ) ) {
 	require dirname( __FILE__ ) . '/vendor/autoload_52.php';
 } else {
 	require dirname( __FILE__ ) . '/vendor/autoload.php';
 }
 
+// Handle any upgrade routines
+if ( is_admin() ) {
```

Comment:
@wpscholar what do you think of making this `is_user_logged_in()`?

Reply:
The logged in part is implied with `is_admin` and I don't want to slow down any front-end requests (e.g. membership sites, e-commerce sites, etc. )